spec-0006/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "SPEC 6 — Keys to the Castle"
+date: 2022-12-19
+author:
+  - "Stéfan van der Walt <stefanv@berkeley.edu>"
+  - "Jarrod Millman <millman@berkeley.edu>"
+discussion:
+endorsed-by:
+---
+
+## Description
+
+<!--
+Briefly and clearly describe the proposal.
+Explain the general need and the advantages of this specific proposal.
+If relevant, include examples of how the new functionality would be used,
+intended use-cases, and pseudo-code illustrating its use.
+-->
+
+Project developers often have to share secrets, such as server passwords and social media logins.
+Typically, such secrets are distributed among those who need them, without any centralized system for tracking the secrets or who has access to them.
+When a server needs to be accessed, there is often a scramble to find someone with credentials.
+This SPEC discusses the requirements for a system to distribute secrets, and provides an example implementation.
+
+## Implementation
+
+### Principles
+
+A system for distributing project secrets must have the following properties:
```

Comment:
I would also add that everything must be documented. E.g. if you add a key, it should at minima say: scope, how to update/revoke/rotate, expiry, comment.